### DISCUSSION TITLE
How to use ImageBrush.Parse ?

### DISCUSSION THREAD

**Discussion by YWilix** [AvaloniaUI/Avalonia#11927] [Q&A] [answered]
Hello , I am trying to understand how to use the ImageBrush.Parse method 
I've tryed using a Relative path  , absolute path but it didn't work
do you have any ideas on how to use it ?

**Comment by maxkatz6** [comment]
I don't think you need to **parse** image from the file path.
I think you need this:
```
new ImageBrush
{
   Source = new Bitmap(path)
}
```

**Reply by YWilix** [reply]
does the path of the bitmap support relative paths or avares ??

**Reply by maxkatz6** [reply, marked as answer]
It's a physical file path. If you need avares, please use AssetsLoader to get the stream and pass it to Bitmap ctor.
